src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -136,38 +134,6 @@ func runClientProxy(cfg *config.Config) {
 	}
 }
 
```

Comment:
This function was moved to `server.go` and `server_stubs.go` so it can be disabled on android hosts based on build tags.